<launch>
  <!-- args that can be set from the command line or a default will be used -->
  <arg name="serial_port" default="/dev/ttyUSB0"/>
  <arg name="serial_baudrate" default="115200"/>
  <arg name="frame_id" default="laser_frame"/>
  <arg name="rotation_speed" default="5"/>

  <node name="sweep_node"	pkg="sweep_ros"  exec="sweep_node">
  	<param name="serial_port"         value="/dev/ttyUSB0"/>
  	<param name="serial_baudrate"     value="115200"/>
  	<param name="frame_id"            value="laser_frame"/>
  	<param name="rotation_speed"	  value="5"/>
  </node>
</launch>
